manager/integration/tests/common.py

```diff
@@ -2348,10 +2393,13 @@ def wait_volume_kubernetes_status(client, volume_name, expect_ks):
     for i in range(RETRY_COUNTS):
         expected = True
         volume = client.by_id_volume(volume_name)
-        ks = volume["kubernetesStatus"]
+        ks = volume.kubernetesStatus
+        ks = json.loads(json.dumps(ks, default=lambda o: o.__dict__))
```

Comment:
Is this still necessary?

Reply:
Yes, `workloadsStatus` object in `volume.kubernetesStatus` is of type `longhorn.RestObject`, and need to be converted to dict to be compared with `expected_ks` value, which of type `dict`

Reply:
But now longhorn.RestObject has __getitem__ for any keys, so `ks[k]` should work directly?

Reply:
it didn't work in this case, the output of `ks[k]` is of type  `longhorn.RestObject` not a `dict`